src/rpcmining.cpp

```diff
@@ -83,25 +81,24 @@ Value getmininginfo(const Array& params, bool fHelp)
     }
 
     obj.push_back(Pair("difficulty",    diff));
-    obj.push_back(Pair("blockvalue",    (uint64_t)GetProofOfWorkReward(0,  GetAdjustedTime(),1)));
+    obj.push_back(Pair("pow_reward",    GetProofOfWorkReward(0,  GetAdjustedTime(),1)/(double)COIN));
     obj.push_back(Pair("netstakeweight2", GetPoSKernelPS2()));
     obj.push_back(Pair("errors",        GetWarnings("statusbar")));
     obj.push_back(Pair("pooledtx",      (uint64_t)mempool.size()));
     //double nCutoff =  GetAdjustedTime() - (60*60*24*14);
-    obj.push_back(Pair("stakeinterest",    (uint64_t)GetCoinYearReward( GetAdjustedTime())));
+    obj.push_back(Pair("pos_interest_year", GetCoinYearReward( GetAdjustedTime())/(double)COIN));
```

Comment:
Could this be named different? pos_interest_year and they see 0.015. might be confusing to new users who don't know its 1.5% per year. maybe pos_interest_year_rate or something more creative then i can be XD

Also haven't been a fan with the whole _ nor - but - would match other getmininginfo. just my opinion . good work. nice to see the wallet being cleaned up and becoming something more